releasenotes/notes/cleanup-timeline-drawer-a6287bdab4459e6e.yaml

```diff
@@ -0,0 +1,27 @@
+---
+upgrade:
+  - |
+    A protected attribute :attr:`_node_start_time` is silently added to
```

Comment:
I wonder if we should have a public getter on this method. If we are going to advertise that people can use it. Basically just add:

```python
@property
def node_start_time(self):
    if not hasattr(self, "_node_start_time"):
        raise AttributeError
    return self._node_start_time
```
(not sure of the best behavior for when it's not set maybe just return `None` I'm not sure)

Then we advertise this public attribute for people to use. Then we can also document it and the format in the class documentation.

Reply:
Thanks. Sounds like this is good idea. Done in https://github.com/Qiskit/qiskit-terra/pull/7935/commits/ff377ab10a894afacc02c47ec570c3fc1b288391